rfcs/text/0024/ticket.yml

```diff
@@ -0,0 +1,151 @@
+---
+- name: ticket
+  group: 2
+  title: Ticket
+  short: Fields describing details of a ticket.
+  description: >
+    A ticket is defined as a documented record of any form of request or reported issue.
+    Tickets can be associated with hardware, software, access, procurement, etc. requests.
+    They can also be associated with ITIL /ITSM support processes
+    Ticket fields provide key basic details about the ticket itself.
+  footnote: >
+    Ticket types include, but are not limited to:
+    Alarm, Event, Incident, Problem, Case, Bug, Feature Request, Task, Epic, User Story,
+    Knowledge Article, Change Request, etc.
+  type: group
+
+  fields:
+
+    - name: assignee
+      level: extended
+      type: keyword
+      short: Assigned to name
+      description: >
+        Name of the person or group with whom the ticket is currently assigned.
+      example: Sector_7-G_Safety_Desk
+
+    - name: created
+      level: core
+      type: date
+      short: Timestamp the ticket was created
+      description: >
+        Timestamp the ticket was created.
+      example: 2021-03-09T17:05:23.773Z
+
+    - name: description.long
+      level: core
+      type: text
+      short: Long description of the ticket
+      description: >
+        Long description / body of the ticket.  Typically contains all details and
+        specifics of the ticket.
+      example: >
+        A fiber cut has occurred in Springfield due to a down telephone pole.  All internet and
+        IP related services are down diddly offline. Customer call in-rates are elevated.
+
+    - name: description.short
```

Comment:
I see the two `description` fields being confusing.

Could `description.short` be a `.summary` field instead, and `description.long` simply `description`?

Reply:
This could certainly be broken out that way.  I see the following all being options:
`.summary` & `.description`
`.title` & `.description`
`.description.short` & `.description.long`

I went with "short" and "long" description simply because that is a common schema definition used for any record where you have both "quick summary" and "full details".